documentation/book/con-partition-reassignment.adoc

```diff
@@ -81,3 +81,32 @@ The following is an example reassignment JSON file that assigns topic `topic-a`,
 
 Partitions not included in the JSON are not changed.
 
+== Reassigning partitions between JBOD volumes
+
+When using JBOD storage in your Kafka cluster, you can also reassign the partitions between specific volumes and their log directories (each volume has one log directory).
+To reassign a partition to a specific volume, the `log_dirs` option has to be added to _<PartitionObjects>_.
```

Comment:
_This is an instruction to do something, so I would use the imperative mood._

To reassign a partition to a specific volume, add the `log_dirs` option to `_<PartitionObjects>_` in the reassignment JSON file.

_Ignore the backticks around PartitionObjects._